### PR TITLE
[SOL-1741 Fix for the Coupon Invoice Django Admin Page]

### PR DESCRIPTION
**Pull request by marjev**
@mjfrey Here's a PR that fixes the broken Coupon Invoice Django Admin page. Please review.

JIRA ticket: [SOL-1741](https://openedx.atlassian.net/browse/SOL-1741)

Here's the screenshot that shows how it looks now:
screencapture-localhost-8002-admin-invoice-invoice-1460723626570

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @marjev! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request.

To automatically create an OSPR issue for this pull request, just visit this link: https://openedx-webhooks.herokuapp.com/github/process_pr?repo=edx%2Fecommerce&number=675

**Comment by clintonb** [comment]
How about this?
1. Remove the order number from the string method. 
2. Create a proper admin with sortable/searchable list display: https://docs.djangoproject.com/en/1.8/ref/contrib/admin/.

As you develop new models, consider making the admin the third thing you do after creating the model and migrations. The admin will be the first place we look when debugging, so it should be fully functional.

**Comment by marjev** [comment]
@clintonb, @mjfrey Below is the new look of the Invoice Django Admin Page. Please take a look at the screenshot below and the code changes. Thanks!

screencapture-localhost-8002-admin-invoice-invoice-1460729638276

File: ecommerce/invoice/admin.py [hunk 1]
```diff
@@ -6,4 +6,5 @@
 
 @admin.register(Invoice)
 class InvoiceAdmin(SimpleHistoryAdmin):
-    pass
+    list_display = ('id', 'basket', 'order', 'business_client', 'state',)
+    search_fields = ['id', 'order__number', 'business_client__name', 'state']
```
Comment (clintonb) [inline ecommerce/invoice/admin.py]: Don't search on state or id. If you have the ID, you can just change the URL, and include the ID. For state, you should use [`list_filter`](https://docs.djangoproject.com/en/1.8/ref/contrib/admin/#django.contrib.admin.ModelAdmin.list_filter).

Comment (marjev) [inline ecommerce/invoice/admin.py]: Done.

**Comment by mjfrey** [comment]
:+1:

**Pull request merged**